canister/src/state.rs

```diff
@@ -52,6 +54,8 @@ pub struct State {
     pub api_access: Flag,
 }
 
+pub const SYNCING_THRESHOLD: u32 = 2;
```

Comment:
docs. I'd also move this constant, along with the `is_fully_synced` into lib.rs, as it's not really part of the state.